src/main/java/com/epam/test_generator/controllers/CaseController.java

```diff
@@ -16,6 +16,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
+import javafx.util.Pair;
```

Comment:
Usage of javafx package in backend application is doubting. I would rather you add dedicated DTO for this task and name it's properties more friendly then just "key" and "value".

Reply:
Before Pair we returned Map, but we had problem with transforming Map into JSON. Because Map looked like a big string and we couldn't parse it, when we wanted check JSON in tests. Then we understood that we can have only one key and we used Pair. Agree that creating a new DTO is the best solution.

Created SuitCaseUpdateDTO and change return type on this DTO.